mfr/server/static/js/mfr.js

```diff
@@ -82,6 +82,7 @@
             self.pymParent.iframe.setAttribute('scrolling', 'yes');
             self.pymParent.iframe.setAttribute('horizontalscrolling', 'yes');
             self.pymParent.iframe.setAttribute('verticalscrolling', 'yes');
+            self.pymParent.iframe.setAttribute('sandbox', 'allow-scripts');
```

Comment:
I think it is ok for us to sandboxing with `allow-scripts`. However, we want to try it out if we can give the iframe a different domain to use the protection from Same Origin Policy (if it is easy and if it breaks nothing).

Reply:
If you don't set `allow-same-origin` it just treats all things as different origin. From my tests we do not need `allow-same-origin` for anything specific to render, so I did not include it.

Am I understanding you right?

Reply:
I misunderstood the `sandbox` attribute. Looks good! 🎆  Let me do a local test.